src/publish-search.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const fs = require('fs-extra');
+const request = require('request');
+const path = require('path');
+const endpoint = process.env.searchEndpoint;
+const token = process.env.token;
+const filePath = path.join(process.cwd(), 'src', 'stache', 'search', 'search.json');
+const errorHandler = require('./error-handler');
+
+function getSearchData() {
+  try {
+    let file = fs.readJsonSync(filePath);
+    return JSON.stringify(file);
+  } catch (error) {
+    return errorHandler(new Error(`[ERROR]: Unable to read search file at ${filePath}! ${error.message}`));
+  }
+}
+
+function publishSearch(argv, config) {
+  if (!(((config || {}).appSettings || {}).stache || {}).search || !fs.existsSync(filePath)) {
```

Comment:
Same as using this above.  I can appreciate the the terseness, but I fear we're gaining it at the expense of readability.